### PR TITLE
Change `::` to `#`

### PR DESCRIPTION
**Pull request by teoyuqi**
This PR changes all occurrences of `::` to `#`

### CODE REVIEW

**Review by cheeheng — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -179,8 +179,8 @@ Given below is an example usage scenario and how the add link mechanism behaves
 **Step 1**: The user decides to add a link to a current module in Plannit using the following input:
 `add-link m/CS1231 l/<link URL> la/<link alias>`.
 
-**Step 2**: The `LogicManager` calls the `LogicManager::execute` method on the user input.
-Then, the `LogicManager` calls the `AddressBookParser::parseCommand` method
+**Step 2**: The `LogicManager` calls the `LogicManager#execute` method on the user input.
```
Comment (ekweirui) [inline docs/DeveloperGuide.md:182]: The formatting should be of the form ‘LogicManager#execute()’ for a method.
The braces are missing.

**Review by ekweirui — COMMENTED** [review]
Looks good, but missing braces for method calls.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (teoyuqi) [reply docs/DeveloperGuide.md]: Good catch! I have added the brackets in for methods.

**Review by ekweirui — APPROVED** [review]
LGTM! Thanks so much for adopting the changes ❤️

**Pull request merged**